### PR TITLE
Frontend SDK Integration

### LINKED ISSUE #227: [Frontend] SDK integration
Atm I'm writing this, the SDK is not ready yet, but once it's ready, a lot of the frontend logic should be removed and replaced using the SDK.

If that suits you @ugur-eren I can assign this one to you.

### PR DESCRIPTION
**Pull request by ugur-eren**
- Constants fully moved to the core and frontend uses the core's constants.
- Safety utils are now used from the core
- useMemecoin hook now uses the useMemecoin hook from the hooks package
- useQuoteToken, useQuoteTokenPrice hooks are imported from the hooks package
- Deploy, Launch, Collect Fees, Extend Liquidty Lock actions are now uses the core's Factory methods

closes #227

### CODE REVIEW

File: packages/frontend/src/pages/Token/LaunchForm/Confirm/Ekubo.tsx [hunk 1]
```diff
@@ -48,106 +49,51 @@ export default function EkuboLaunch({ previous }: LastFormPageProps) {
...
+  const launch = useCallback(async () => {
+    if (!memecoin || !startingMcap || !quoteToken || !hodlLimit || !ekuboFees) return
+
+    // Sort by index and map to address and parsed amount
+    const teamAllocations = Object.entries(teamAllocation)
+      .sort(([a], [b]) => +a - +b)
+      .map(([, holder]) => ({
+        address: holder.address,
+        amount: parseFormatedAmount(holder.amount),
+      }))
```
Comment (0xChqrles) [inline packages/frontend/src/pages/Token/LaunchForm/Confirm/Ekubo.tsx:65]: why not just:

```ts
Object.values(teamAllocation)
      .filter(Boolean)
      .map((holder) => ({
        address: holder.address,
        amount: parseFormatedAmount(holder.amount),
      }))
```

?

File: packages/frontend/src/pages/Token/LaunchForm/Confirm/StandardAmm.tsx [hunk 2]
```diff
@@ -61,80 +56,55 @@ export default function StarndardAmmLaunch({ previous, amm }: StarndardAmmLaunch
...
+  const launch = useCallback(async () => {
+    if (!memecoin || !startingMcap || !quoteToken || !hodlLimit) return
+
+    // Sort by index and map to address and parsed amount
+    const teamAllocations = Object.entries(teamAllocation)
+      .sort(([a], [b]) => +a - +b)
+      .map(([, holder]) => ({
+        address: holder.address,
+        amount: parseFormatedAmount(holder.amount),
+      }))
```
Comment (0xChqrles) [inline packages/frontend/src/pages/Token/LaunchForm/Confirm/StandardAmm.tsx:72]: same as above

**Review by 0xChqrles — CHANGES_REQUESTED** [review]
Huge thank you for this PR ! I've Just raised 2 minor comments

File: packages/frontend/src/pages/Token/LaunchForm/Confirm/Ekubo.tsx (hunk 1 above)
Comment (ugur-eren) [reply packages/frontend/src/pages/Token/LaunchForm/Confirm/Ekubo.tsx:65]: I thought team allocation index matters, so sorted by it. Removed the sort and map directly.

**Comment by ugur-eren** [comment]
Hi @0xChqrles I've resolved your comments, Thanks!

**Review by 0xChqrles — APPROVED** [review]
lgtm

**Pull request merged**